### ISSUE TITLE
Maintain selection of multiple tabs when using the "Copy URLs" functionality

### ISSUE THREAD

**Issue by MNeMoNiCuZ** [mbnuqw/sidebery#1297]
## Description
When I copy the URLs of several tabs, I find it unfortunate that the tabs I had selected gets deselected after doing this action.
When I do this, I want to copy the URLs, and then close all the tabs as I have their URLs.

If implemented, please make it an option, as I can see others liking it the way it is.
This could also be considered for other right-click features too, such as Copy Titles, Unload, Move To. Maybe some others.

**Comment by rieje** [comment]
Agreed, I often want to perform another action after Copy URLs but since selection gets lost, I need to manually select them again. If I'm selecting from these tabs from throughout the tab tree, it's not feasible to go back and select the same tabs so a workaround is to first create a temporary tab group, move them selected tabs there, then perform actions by selecting all the tabs there, then move the tabs back to their original group and then delete the temporary tab group. This workaround is quite involved and also doesn't preserve the position of the tabs.
